src/get-started/updating-your-code/index.md.njk

```diff
@@ -1,24 +1,16 @@
 ---
 layout: layout-pane.njk
-title: Updating from GOV.UK Elements and Frontend Toolkit
+title: Replace our old variables, functions and mixins
```

Comment:
"our old" sounds a bit odd, would something like "legacy" work better?

Reply:
Hi @hannalaakso! We had some specific feedback at 2i that 'old' was the better term and works ok on its own. And now that we'll be updating the old repos to say "You should use the Design System", it feels we can be comfortable describing these kinds of things as 'old' now. What do you think?

Reply:
Could people think that 'old' means previous versions of the Design System e.g. migrating from version 1.0.0 to 2.0.0

Reply:
Yeah, happy with 'old repos' or 'old frameworks' but I think 'old variables, functions and mixins' may not be clear by itself.

Reply:
Sorry, only just saw this feedback! I've updated the title - let me know if it all looks ok.